### PR TITLE
Update CHANGES.md for release 0.24.0

### PR DESCRIPTION
**Pull request by noahl**
Release 0.24. The only change to master is updating CHANGES.md.

### CODE REVIEW

**Comment by noahl** [comment]
@ehankland @voellm Please take a look.

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+
+New features:
+* CS2 rachel web serving benchmark (GH-451, GH-605)
+* Add [AliCloud](http://intl.aliyun.com/) provider (GH-611)
+* Add tomcat/wrk benchmark (GH-598)
+
+Breaking changes:
+* Remove `--parallelism` flag (GH-633)
+* Rename `fio` flags (GH-518, GH-581)
+* (Future breaking change) Rename disk types and disk-related flags (GH-599)
```
Comment (voellm) [inline CHANGES.md]: Might want to change this to "Added new disk types and options to clarify use."
(breaking change) we will remove the current options in a coming release.

**Comment by voellm** [comment]
Talked thought some changes as well.  Once those are in LGTM

**Comment by cmccoy** [comment]
test this please

**Comment by noahl** [comment]
I just uploaded the changes I discussed with Tony.

**Comment by cmccoy** [comment]
test this please

**Pull request merged**